azure-pipelines.yml

```diff
@@ -38,26 +38,28 @@ stages:
     strategy:
       matrix:
         linux:
-          demandsName: ImageOverride -equals Build.Ubuntu.2204.Amd64.Open
+          demandsName: 1es-ubuntu-2004-open
```

Comment:
the demands weren't being respected for some reason when we put the entire string in here.  So everything was actually just running on windows...

Now we actually run on the correct platform.

Reply:
This also uses 1es-ubuntu because it has more of the packages we need (libgtk, libgbm, xvfb)